### ISSUE TITLE
Please, make real tags for bower support

### ISSUE THREAD

**Issue by kud** [cowboy/jquery-hashchange#37]
Like `1.3.0` no `v` or no just two digits.

**Comment by cowboy** [comment]
I've made all my tags starting with "v" forever. It simplifies shell auto-completion for me. It's also a fairly common practice, and as such bower should support it. Please file an issue with bower to support this very, very common behavior.

**Comment by kud** [comment]
I'll do a report to bower. And yes, `v` is okay but you miss a digit though. cf http://semver.org/

**Comment by MoOx** [comment]
Yep, should be something like v1.2.0 according to bower/semver spec

**Comment by satazor** [comment]
Yes v is fine, but not two digits. That's an invalid semver format.

**Comment by cowboy** [comment]
FWIW, semver wasn't nearly as popular when I wrote this plugin as it is now, but I'll add this to my list of things to do. Thanks, all.
